README.md

```diff
@@ -15,17 +15,17 @@ Build with `xmtp-js` to provide messaging between blockchain wallet addresses, d
 
 `xmtp-js` was included in a [security assessment](https://xmtp.org/assets/files/REP-final-20230207T000355Z-3825cbc68c115f4ec81f3b1d53a24fce.pdf) prepared by [Certik](https://www.certik.com/company/about).
 
-To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+To learn more about XMTP and get answers to frequently asked questions, see the [XMTP documentation](https://xmtp.org/docs).
 
-## Example apps built with `xmtp-js`
+## Quickstart and example apps built with `xmtp-js`
 
-- [XMTP Quickstart React app](https://github.com/xmtp/xmtp-quickstart-react): Provides a basic messaging app demonstrating core capabilities of the SDK. The app is intentionally built with lightweight code to help make it easier to parse and start learning to build with XMTP.
+- Use the [XMTP React quickstart app](https://github.com/xmtp/xmtp-quickstart-react) as a tool to start building an app with XMTP. This basic messaging app has an intentionally unopinionated UI to help make it easier for you to build with.
```

Comment:
I actually think we should link to the react playground instead here. And soon also the vue playground. Since both of those are backed directly with the JS SDK